aptos-move/framework/aptos-framework/sources/code.move

```diff
@@ -148,6 +148,19 @@ module aptos_framework::code {
         publish_package(owner, util::from_bytes<PackageMetadata>(pack_serialized), code)
     }
 
+    /// Same as `publish_package_txn` but allows to split the metadata into multiple parts for working around
```

Comment:
Is this better to support a vector chunks? chunks: vector<vector<u8>>

also nit: chunck => chunk

Reply:
I mentioned that in the channel and it is also documented in the code: the size of constants in Move is restricted to 64k. The metadata can be larger than that. You can not put into `vector[vector[..` because the compiler seems to make via constant folding a single constant out of this. Therefore we have to make it explicit via `chunkN` variables.

Fixed the spelling.

Reply:
Hmm I see. That makes sense then. What happens if one of the chunks is empty. Would that still work?

Reply:
Yes they can be empty, but this would be unlikely because the overall metadata is divided into 3 equally sized chunks.